android/build.gradle

```diff
@@ -18,12 +18,13 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
+    namespace 'com.cotecna.capacitor.mediastore'
     compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 32
     defaultConfig {
-        minSdkVersion project.hasProperty('minSdkVersion') ? rootProject.ext.minSdkVersion : 22
+        minSdkVersion project.hasProperty('minSdkVersion') ? rootProject.ext.minSdkVersion : 29
         targetSdkVersion project.hasProperty('targetSdkVersion') ? rootProject.ext.targetSdkVersion : 32
-        versionCode 1
-        versionName "1.0"
```

Comment:
We should increase the iOS version too because the upgrade of Capacitor 6 in the package affects both platforms